components/constellation/constellation.rs

```diff
@@ -133,6 +134,9 @@ pub struct Constellation<Message, LTF, STF> {
     /// to receive sw manager message
     swmanager_receiver: Receiver<SWManagerMsg>,
 
+    /// A map from top-level frame id and registered domain name to script channels.
```

Comment:
Let's add a comment to the effect of "This double indirection ensures that separate tabs do not share script threads, even if the same domain is loaded in each."